### ISSUE TITLE
Instruction issue in 'Get Geolocation Data' waypoint

### ISSUE THREAD

**Issue by Salvero** [FreeCodeCamp/FreeCodeCamp#4068]
## Waypoint: Get Geolocation Data

I see that we can't read properly numbers in both latitude and longitude, here's screenshot what it looked like below.  

See right side in mobile view. 
![screen shot 2015-10-31 at 12 45 16 pm](https://cloud.githubusercontent.com/assets/9256193/10864706/6c27bec4-7fcd-11e5-95f4-007c86915659.png)

I think, we can make it more readable and better visible for other new campers.  

My Code: 

```javascript
  if (navigator.geolocation) {
    
  navigator.geolocation.getCurrentPosition(function(position){
    
  $("#data").html("Latitude: " + position.coords.latitude + "<br/>" + "Longitude: " + position.coords.longitude);
    });  
  }
```
![screen shot 2015-10-31 at 12 49 25 pm](https://cloud.githubusercontent.com/assets/9256193/10864727/e10f3c76-7fcd-11e5-8354-599b2eac5bdc.png)

**Comment by ltegman** [comment]
Thanks for the report, but this issue was resolved by PR #3986 and is pending deployment to the website.

Closing as resolved, hopefully this update will hit the website soon.

Thanks and happy coding!

**Comment by Salvero** [comment]
Oh Thanks for quick response! :smile:
